dspace/modules/api/src/main/java/org/datadryad/anywhere/AssociationAnywhere.java

```diff
@@ -168,8 +182,8 @@ private static Document loadCustomerInfo(String customerId)
      * @return status of response.
      * @throws AssociationAnywhereException
      */
-    public static String deductCredit(String customerId) throws AssociationAnywhereException {
-        log.debug("deducting one credit for customerId " + customerId);
+    public static String tallyCredit(Context context, String customerId, String dataPackageID) throws AssociationAnywhereException {
```

Comment:
Two things here: 
1. please update the comments above the method since this has been renamed.
2. In ReauthorizationCreditActionXMLUI, only DOIs are used as dataPackageID: is this always the case? If so, perhaps renaming the parameter to dataPackageDOI would be clearer.